sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -215,7 +215,22 @@ export class EventHubReceiver extends LinkEntity {
           this.runtimeInfo
         );
       }
-      this._onMessage!(evData);
+      try {
+        this._onMessage!(evData);
+      } catch (err) {
+        // This ensures we call users' error handler when users' message handler throws.
+        if (!isAmqpError(err)) {
+          log.error(
+            "[%s] An error occurred while running user's message handler for the message " +
+              "'%s' on the receiver '%s': %O",
+              this._context.connectionId,
+              evData.body,
```

Comment:
Is it a common practice we have to log the message body in the logs?

Reply:
No, I added that, because body is the only required parameter of the message.